docs/issue-labels.rst

```diff
@@ -7,93 +7,89 @@ what they stand for.
 .. _GitHub issue tracker: https://github.com/rtfd/readthedocs.org/issues
 
 *Bug*
-    An issue describing unexpected or malicious behaviour of the
-    readthedocs.org software.
+    An issue describing unexpected or malicious behaviour of the readthedocs.org
+    software. A Bug issue differs from an Improvement issue in that Bug issues
+    are given priority on our roadmap. On release, these issues generally only
+    warrant incrementing the patch level version.
 
 *Community Effort*
-    Tickets with this label are valid issues that the core team thinks are
+    Issues with this label are valid issues that the core team thinks are
     worth to fix or implement in the future. However the core team's resources
-    are too scarce to address these issues. Tickets marked with this label
+    are too scarce to address these issues. Issues marked with this label
     are issues that the core team will **not** work on, but contributions
     from the community are very welcome.
 
 *Design*
     Issues related to the UI of the readthedocs.org website.
 
-*Enhancement*
-    Feature requests and ideas about how to make readthedocs.org better in
-    general will have this label.
-
-*Feature Overview*
-    Features that are too big to be tackled in one ticket are split up into
-    multiple tickets. A feature overview ticket is then explaining the overarching
-    idea. See the milestone of the feature overview ticket for all related
-    tickets.
+*Feature*
+    Issues that describe new features will be given this label **only** once the
+    feature is an accepted feature on our roadmap. This label does not describe
+    issues that may become features some day. Issues that do not describe
+    new features, such as code cleanup or fixes that are not related to a bug,
+    should probably be given the Improvement label instead. On release, issues
+    with the Feature label warrant at least a minor version increase.
```

Comment:
I'd be more explicit here, and call it an "Accepted Feature", to make the intent more clear.

Reply:
I noted before this, maybe an `Accepted` tag would be another addition. Does that add too much variability to issue state? We'd have:

* `Accepted` + `Feature` -- makes sense
* `Accepted` + `Improvement` -- also makes sense
* `Accepted` + `Bug` -- i guess this makes sense too, we do have some bugs in a non-accepted sort of state. They are off our roadmap and/or not replicated.

Reply:
Yea, I think `Accepted Bug` could also be verified/triaged, as well as "something we want to fix".